src/c/core/session/session.c

```diff
@@ -626,14 +626,14 @@ void uxr_flash_output_streams(
 //==================================================================
 //                             PRIVATE
 //==================================================================
-bool uxr_acknack_pong(
+pong_status_t uxr_acknack_pong(
         ucdrBuffer* buffer)
 {
     bool success = false;
     bool ret = false;
```

Comment:
```suggestion
    pong_status_t ret = NO_PONG_STATUS;
```
Maybe we could simplify the return statement